src/main/java/org/alliancegenome/curation_api/services/validation/dto/fms/GeneExpressionAnnotationFmsDTOValidator.java

```diff
@@ -118,7 +104,7 @@ public GeneExpressionAnnotation validateAnnotation(GeneExpressionFmsDTO geneExpr
 
 		ObjectResponse<AnatomicalSite> anatomicalSiteObjectResponse = validateAnatomicalSite(geneExpressionFmsDTO);
 		if (anatomicalSiteObjectResponse.hasErrors()) {
-			response.addErrorMessage("whereExpressed", anatomicalSiteObjectResponse.errorMessagesString());
+			response.addErrorMessage("expressionPattern", anatomicalSiteObjectResponse.errorMessagesString());
```

Comment:
This change doesn't make sense to me

Reply:
With that change the errors will be like "expressionPattern - whereExpressed - error message"